src/tools/immutable_object_cache/Types.cc

```diff
@@ -17,7 +17,7 @@ ObjectCacheRequest::ObjectCacheRequest(uint16_t t, uint64_t s)
 ObjectCacheRequest::~ObjectCacheRequest() {}
 
 void ObjectCacheRequest::encode() {
-  ENCODE_START(1, 1, payload);
+  ENCODE_START(2, 1, payload);
```

Comment:
@dillaman Is this the change you expect? also "DECODE_START(2, i);" and through "if (i.end())"  downward compatibility